src/components/TeamMemberStyle.js

```diff
@@ -0,0 +1,102 @@
+import { makeStyles } from '@material-ui/core/styles';
+const styles =makeStyles(theme => ({
+        heading: {
+            fontFamily: "'Catamaran', sans-serif",
+            textTransform: "uppercase",
+            color: '#000000',
+            textAlign: "center",
+        },
+        root: {
+            display: 'flex',
+            flexWrap: 'wrap',
+        },
+        "grid-container": {
```

Comment:
makeStyles takes a JSON object. So, you can use gridContainer in place of grid-container. Makes it uniform with others.

Reply:
No i thought we should use kebab case.So i went for it